metabase/metabase
# Improve visuals for VisualizationError

Epic https://github.com/metabase/metabase/issues/26837

How to test:
- New -> SQL Query
- `SELECT {{a}}` and run the query
- You should see the error message about missing parameter

<img width="1372" alt="Screenshot 2022-12-12 at 14 35 09" src="https://user-images.githubusercontent.com/8542534/207046852-ddf531e6-3828-4a4f-bcd4-579eef0e50ea.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27159)
